Src/applications/dronecan/CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ include(${SRC_DIR}/modules/dronecan/pwm/CMakeLists.txt)
 
 if(USE_PLATFORM_NODE_V3)
     include(${SRC_DIR}/modules/imu/CMakeLists.txt)
+    include(${SRC_DIR}/modules/dronecan/canopen/CMakeLists.txt)
```

Comment:
Please, remove canopen here. It is just an example and should not be included in the default build.